app.py

```diff
@@ -137,6 +137,7 @@ def github_team_members(client=None, owner=None, team_id=None, attribute="userna
     :return: team_members
     :rtype: list
     """
+    ignore_users = [u for u in os.getenv('IGNORE_USERS', '').split(' ')]
```

Comment:
@kristian-lesko putting this into the environment would be somewhat limiting (which may or may not be what we want). In this case, `IGNORE_USERS` would be globally ignored for all teams, and would limit the number of ignored users to being what you could reasonably fit into a single line in your `.env`. We currently use `syncmap.yml` to create custom maps for teams that don't match directory names. It may make sense to have this be a bit more extensible and add `ignore_users: []` there.

```yaml
---
mapping:
  - github: demo-team
    directory: ldap super users
  - github: demo-team-2
    directory: another-group

ignore_users:
  - userA
  - userB
  - userC
```

This opens possibilities to have specific users ignored in specific teams:
```yaml
...
ignore_users:
  - team: teamA
    users:
      - userA
      - userB
```

This may end up being more complicated than we want... Can you see a scenario where granular ignores would be useful? Even without granular ignores I think the `YAML` would be a better place to put these.

What do you think?

Reply:
@primetheus thank you for the review; I agree that the `syncmap.yml` will be a better place for this kind of config.

Personally, I don't see much value in having per-team ignore lists. For our specific use case, we need to ignore users that are present in a GitHub organization but not in a directory; lower granularity is not needed.

Let me try to edit the PR to reflect the first option you mentioned (global `ignore_users` with a single list in the YAML config).